inkscape_driver/eggbot_reorder.py

```diff
@@ -30,60 +32,61 @@
 def dist( x0, y0, x1, y1 ):
 	return math.sqrt( ( x1 - x0 ) ** 2 + ( y1 - y0 ) ** 2 )
 
-def find_ordering_naive( objlist ):
+def find_ordering( objlist, allowReverse ):
 	"""
 	Takes a list of (id, (startX, startY, endX, endY)), and finds the best ordering.
-	Doesn't handle anything fancy, like reversing the ordering, but it's useful for now.
-	Returns a list of JUST THE IDs, in a better order, as well as the original and optimized
-	"air distance" which is just the distance traveled in the air. Perhaps we want to make
-	these comparison distances into something more relevant such as degrees traveled?
+	Uses a greedy algorithm which can reverse the path direction if necessary
+	Returns a list of (id, reverse), as well as the original and optimized
+	"air distance" which is just the distance traveled in the air. Reverse indicate if the path must be reversed
```

Comment:
A param name of "allowReverse" seems to conflict with the description here of "must be reversed". How does this work?

Reply:
If `allowReverse==False`, all the `reverse` booleans will be False.